ui-packages/packages/trusty/src/components/Organisms/CounterfactualOutcomeSelection/CounterfactualOutcomeSelection.scss

```diff
@@ -1,20 +1,34 @@
+.counterfactual__outcomes-alert {
+  margin-bottom: var(--pf-global--spacer--sm);
+}
+
 .counterfactual__outcomes-grid {
   width: 100%;
   display: inline-grid;
-  grid-template-columns: repeat(2, 1fr);
+  grid-template-columns: 1fr 2fr;
   grid-column-gap: var(--pf-global--spacer--lg);
 
+  border-top: var(--pf-global--palette--black-300);
+  border-top-style: solid;
+  border-top-width: 1px;
```

Comment:
If you prefer a shorter syntax you could set them all with the shorthand property:
`border-top: 1px solid var(--pf-global--palette--black-300);`

Reply:
@kelvah Updates completed.